frontend/src/scss/PrimeStyles.scss

```diff
@@ -839,20 +839,6 @@ $results-dropdown-spacing: #{units(4)} - #{units(2)} - 22px - #{units(4)}; // he
   white-space: nowrap;
 }
 
-// toast
```

Comment:
⚠️ moved to its own file here: frontend/src/app/commonComponents/SRToastContainer.scss